.template/spec/variants/web/tool_versions_spec.rb

```diff
@@ -0,0 +1,7 @@
+describe 'Web variant - .tool-versions' do
+  subject { file('.tool-versions') }
+
+  it 'pins the nodejs version' do
```

Comment:
I am not sure of the wording `pin` (even in the context of a test) because of the meaning it has in Rails 7.x with `importmap-rails` (see [docs](https://github.com/rails/importmap-rails/)). How about instead:

```suggestion
  it 'sets the Node.JS version' do
```